libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -690,6 +691,40 @@ void test_MQTT_Connect_receiveConnack( void )
     MQTT_DeserializeAck_ExpectAnyArgsAndReturn( MQTTBadResponse );
     status = MQTT_Connect( &mqttContext, &connectInfo, NULL, timeout, &sessionPresent );
     TEST_ASSERT_EQUAL_INT( MQTTBadResponse, status );
+
+    /* Same set of tests with retries. MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT is 2*/
```

Comment:
If it's the same set of tests, I think we should put them in a new function: `test_MQTT_Connect_receiveConnack_retries`